composer.json

```diff
@@ -7,7 +7,7 @@
         "php": ">=5.4.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4|^5",
+        "phpunit/phpunit": "^4.8|^5.7",
```

Comment:
Well, it's not entirely unrelated, as we do require a newer version than was present for the new test. Anyway, I just fixed the skip test and now we require PHPUnit 4.8.13 anyway, see commit message for an explaination.